StringArt/StringArt.js

```diff
@@ -0,0 +1,128 @@
+import { Polar } from "../sketchlib/Polar.js";
+
+const MODULUS = 60;
+
+/**
+ * Polynomial ax^3 + bx^2 + cx + d where the coefficients
+ * and values are evaluated mod n
+ */
+class Polynomial {
+  constructor(a, b, c, d) {
+    this.coefficients = [a, b, c, d];
+  }
+
+  compute(x) {
```

Comment:
Test cases:

- Negative inputs should be handled gracefully
- 0 maps to d
- 1 maps to the sum of coefficients
- hand-compute all the values for a particular polynomial and verify that each input produces the corresponding output